### ISSUE TITLE
Button no longer works for stories

### ISSUE THREAD

**Issue by LikeaDize** [y252328/Instagram_Download_Button#39]
Button no longer appears on IG stories, but seems to work on posts. Also Story Highlights appear broken.

**Comment by MickeyM007** [comment]
> For a workaround, you can replace the storySelector with "div.x1q0g3np:nth-child(2) ._ac0m"

how?

**Comment by embeddedjunkie** [comment]
@MickeyM007 hello, check greasyfork again, a new update just pushed to fix stories. 1.17.5

**Comment by MickeyM007** [comment]
> @MickeyM007 hello, check greasyfork again, a new update just pushed to fix stories. 1.17.5

Yeah I saw it, thank you!! I updated, but now only stories are working (firefox), I already placed a comment.

**Comment by MickeyM007** [comment]
Just so you know, I'm on firefox.. When I enable v1.17.3 and v1.17.5 at the same time, everything works... so far.

**Comment by Antiever** [comment]
No button in chromium.
1.17.6
